test/liquid_voting/delegations_test.exs

```diff
@@ -38,6 +38,8 @@ defmodule LiquidVoting.DelegationsTest do
       ]
     end
 
+    @proposal_url "https://www.someorg/proposalX"
```

Comment:
Usually we either have setup contexts or module attributes like this, but not both at the same time on the same test file. So it's clear where all the test data is. I'd just add this to the setup context in this case

Reply:
Do you mean I should change line 14 to: `proposal_url = @proposal_url` and leave line 41 as is? Or something else?

I need a way to sett the proposal url in lines like line 88: `original_delegation = insert(:delegation, proposal_url: @proposal_url)` Where I am trying to avoid specifying the same url string in multiple tests (though I don't know why that matters really)

Reply:
So, I removed the attribute `@proposal_url =  "https://www.someorg/proposalX"` and edited the 3 occurrences of its use in tests to: `original_delegation = insert(:delegation, proposal_url: "https://www.someorg/proposalX")`.  
This doesn't feel like what you meant, though.

Reply:
I mean adding it to the setup so you can access it with `context[:proposal_url]` :)

Reply:
Done. Thanks for clarifying.